src/plugins/controls/public/control_types/time_slider/time_slider.tsx

```diff
@@ -77,6 +77,7 @@ export const TimeSlider: FC<TimeSliderProps> = ({
 
   return (
     <Component
+      id={id}
```

Comment:
What is the ID used for here?

Reply:
It was necessary for the time-slider functional tests, which are now commented out 😆 - specifically, I needed the components to have their ID for when I call `dashboardControls.verifyControlType(controlId, <type>)`. But I think it's worth keeping that in for when we eventually add the time slider back in - all other controls have their `id` as part of their relative `component`.

Reply:
Sounds good!